api-tests/apply_route_test.go

```diff
@@ -115,11 +115,11 @@ func TestApplyPgShouldReturnSameMajorVersion(t *testing.T) {
 	pgParams := &version_service.VersionServiceApplyParams{
 		Apply:           "latest",
 		OperatorVersion: "1.0.0",
-		Product:         "postgresql-operator",
+		Product:         "pg-operator",
 	}
 	pgParams.WithTimeout(2 * time.Second)
 
-	for _, v := range []string{"11.0", "12.0", "13.0"} {
+	for _, v := range []string{"12.8", "13.4"} {
```

Comment:
we need to add 14.1, 13.5 and 12.9 (for next release)

Reply:
Let's add it on exact release time